File: src/PowerChat.Application/Channels/Models/MessageModel.cs
using System;

namespace PowerChat.Application.Channels.Models
{
    public class MessageModel
    {
        public long Id { get; set; }
        public long AuthorId { get; set; }
        public string Content { get; set; }
        public DateTime SentDate { get; set; }
        public DateTime? Seen { get; set; }
        public bool Own { get; set; }
    }
}

File: src/PowerChat.Mobile/src/models/chat/MessageModel.js
export class MessageModel {
  constructor(id, authorId, content, sentDate, seen, own) {
    this.id = id;
    this.authorId = authorId;
    this.content = content;
    this.sentDate = sentDate;
    this.seen = seen;
    this.own = own;
  }
}